rero_ils/modules/documents/jsonschemas/documents/document-v0.0.1_src.json

```diff
@@ -0,0 +1,925 @@
+            "title": "Name",
+            "description": "Person's or organisation's name.",
+            "type": "string"
+          },
+          "type": {
+            "title": "Type",
+            "description": "Identify if the author is a person or an organisation.",
+            "type": "string",
+            "enum": [
+              "person",
+              "organisation"
+            ],
+            "default": "person"
+          },
+          "date": {
+            "title": "Date",
+            "description": "Information about the birth and the death of a person. Helpful to disambiguate people.",
+            "type": "string"
+          },
+          "qualifier": {
+            "title": "Qualifier",
+            "description": "Information about the person, ie her profession. Helpful to disambiguate people.",
+            "type": "string"
+          }
+        }
+      }
+    },
+    "copyrightDate": {
+      "title": "Copyright Date",
+      "description": "Copyright Date of the resource.",
+      "type": "array",
+      "minItems": 1,
+      "items": {
+        "type": "string",
+        "pattern": "^([\u00a9\u2117]\\s)?\\d{4}.*",
+        "minLength": 4
+      }
+    },
+    "provisionActivity": {
+      "title": "Provision Activity",
+      "description": "Provision Activity of the resource.",
+      "type": "array",
+      "minItems": 1,
+      "items": {
+        "type": "object",
+        "additionalProperties": false,
+        "properties": {
+          "type": {
+            "title": "Type",
+            "type": "string",
+            "enum": [
+              "bf:Publication",
+              "bf:Manufacture",
+              "bf:Distribution",
+              "bf:Production"
+            ]
+          },
+          "statement": {
+            "title": "Statement of od place and/or agent of publication",
+            "description": "Statement of place and/or agent of the resource.",
+            "type": "array",
+            "items": {
+              "type": "object",
+              "properties": {
+                "type": {
+                  "title": "Type",
+                  "type": "string",
+                  "enum": [
+                    "bf:Place",
+                    "bf:Agent"
+                  ]
+                },
+                "canton": {
+                  "title": "Canton",
+                  "description": "Canton.",
+                  "type": "array",
+                  "minItems": 1,
+                  "items": {
+                    "type": "string",
+                    "$ref": "#/definitions/canton"
+                  }
+                },
+                "country": {
+                  "title": "Country",
+                  "description": "Country.",
+                  "type": "string",
+                  "$ref": "#/definitions/country"
+                },
+                "label": {
+                  "title": "Label",
+                  "description": "Provision Activity place and/or agent.",
```

Comment:
Is this `description` actually the one we want? Should it be removed?

Reply:
Don't know

Reply:
Ok I delete it!